tests/conftest.py

```diff
@@ -64,6 +64,11 @@ def gordo_project():
     return tu.GORDO_PROJECT
 
 
+@pytest.fixture(scope="session")
+def gordo_revision():
+    return tu.GORDO_REVISION
```

Comment:
Would be good as a fixture to follow suite with the rest of the test project configuration.